Sources/SwiftDocC/LinkTargets/LinkDestinationSummary.swift

```diff
@@ -254,23 +258,94 @@ extension LinkDestinationSummary {
     ///   - path: The bundle-relative path to this page.
     ///   - taskGroups: The task groups that lists the children of this page.
     ///   - compiler: The content compiler that's used to render the node's abstract.
-    init(documentationNode: DocumentationNode, path: String, taskGroups: [TaskGroup], platforms: [PlatformAvailability]?, compiler: inout RenderContentCompiler) {
-        let symbol = documentationNode.semantic as? Symbol
+    init(documentationNode: DocumentationNode, path: String, taskGroups: [TaskGroup], variantTaskGroups: [[RenderNode.Variant.Trait]: [TaskGroup]], platforms: [PlatformAvailability]?, compiler: inout RenderContentCompiler) {
+        let redirects = (documentationNode.semantic as? Redirected)?.redirects?.map { $0.oldPath }
+        let referenceURL = documentationNode.reference.url
+        
+        guard let symbol = documentationNode.semantic as? Symbol, let summaryTrait = documentationNode.availableVariantTraits.first(where: { $0.interfaceLanguage == documentationNode.sourceLanguage.id }) else {
+            // Only symbol documentation currently support multi-language variants (rdar://86580915)
+            self.init(
+                kind: documentationNode.kind,
+                language: documentationNode.sourceLanguage,
+                path: path,
+                referenceURL: referenceURL,
+                title: ReferenceResolver.title(forNode: documentationNode),
+                abstract: (documentationNode.semantic as? Abstracted)?.renderedAbstract(using: &compiler),
+                availableLanguages: documentationNode.availableSourceLanguages,
+                platforms: platforms,
+                taskGroups: taskGroups,
+                usr: nil,
+                declarationFragments: nil,
+                redirects: redirects,
+                variants: []
+            )
+            return
+        }
+        
+        // Precompute the summarized elements information so that variants can compare their information against it and remove redundant duplicate information.
+        
+        // Multi-language symbols need to access the default content via the variant accessors (rdar://86580516)
+        let kind = DocumentationNode.kind(forKind: (symbol.kindVariants[summaryTrait] ?? symbol.kindVariants[.fallback] ?? symbol.kind).identifier)
```

Comment:
If the behavior is the same, I would prefer to call `symbol.kind` (and `symbol.title`, etc.) so that this code behaves consistently with how those properties are computed (in case that ever changes). That also helps avoid mistakes with force-unwrapping `first` when the symbol convenience property is optional.